backup-tools/repl-image/files/replication-run.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
-chmod 600 /backup/env/*
-cron -f
+while true; do
+  /usr/local/sbin/backup-replication.sh
+  sleep 14400
```

Comment:
14400 seconds = 4 hours... that would not match our intervals... all 5 mins = 300 seconds where perfekt.. or when need all 10 mins (600)